docs/units-and-global-variables.rst

```diff
@@ -73,6 +73,18 @@ Block and Transaction Properties
     This includes calls to library functions.
 
 .. note::
+    Do not rely on ``block.timestamp`` and ``block.blockhash`` as a source of randomness,
```

Comment:
Please also mention `now`

Reply:
It is in the list above this note, but can add it.